### PR TITLE
Ammar Vehbi - Homework Week 3

### PR DESCRIPTION
**Pull request by ammarvehbi**
**You can run the project by one of the commands below**

1. `npm run server` (runs the project manually from server using express, project will run on http://localhost:5000/)
2. `npm run client` (runs the project automatically using react, project will run on http://localhost:3000/)
3. `npm run dev` (runs the project both using react and server, project will run both on http://localhost:5000/ and http://localhost:3000/)

### CODE REVIEW

File: homework-submission/week3/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# React Week 3 Homework - Ammar Vehbi
```
Comment (paulbremer) [inline homework-submission/week3/README.md]: Updating the readme, very good!

File: homework-submission/week3/client/src/App.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import HomeworkWeek3 from './components/HomeworkWeek3/index';
+
+class App extends React.Component {
+  state = {
+    data: [],
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
```
Comment (paulbremer) [inline homework-submission/week3/client/src/App.js]: What if the request fails? Make sure you add a `catch` for the errors.

File: homework-submission/week3/client/src/components/HomeworkWeek3/style.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0 auto;
+  padding: 10px 15px;
+  text-align: left;
+  background: rgba(255, 255, 255, 0.1);
+  display: table;
```
Comment (paulbremer) [inline homework-submission/week3/client/src/components/HomeworkWeek3/style.css]: `display: table`? We're not really using table layout here, flexbox would work fine in this case.

File: homework-submission/week3/README.md (hunk 1 above)
Comment (ammarvehbi) [reply homework-submission/week3/README.md]: :pray:

File: homework-submission/week3/client/src/App.js (hunk 2 above)
Comment (ammarvehbi) [reply homework-submission/week3/client/src/App.js]: Yeah, I forgot to handle the errors :pray: done.

File: homework-submission/week3/client/src/components/HomeworkWeek3/style.css (hunk 3 above)
Comment (ammarvehbi) [reply homework-submission/week3/client/src/components/HomeworkWeek3/style.css]: Done :pray:

**Pull request closed without merging**